src/vs/workbench/contrib/search/common/constants.ts

```diff
@@ -50,3 +50,5 @@ export const FileMatchOrFolderMatchWithResourceFocusKey = new RawContextKey<bool
 export const FileFocusKey = new RawContextKey<boolean>('fileMatchFocus', false);
 export const FolderFocusKey = new RawContextKey<boolean>('folderMatchFocus', false);
 export const MatchFocusKey = new RawContextKey<boolean>('matchFocus', false);
+
+export const OneLineHeight = 24;
```

Comment:
You can leave this in searchWidget.ts, it doesn't really apply to all of "search"

Reply:
alright thanks